CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Change Log
 
-## 0.0.9
```

Comment:
why was 0.0.9 removed? wasn't it released?

Reply:
released but with no interesting change to users, it was an external dependencies which was updated which allowed to publish on open-vsx